scripts/beelievers_auction_finalize.ts

```diff
@@ -94,28 +88,32 @@ export function batchAddresses(addresses: string[], slot: number): string[][] {
 	return _.chunk(addresses, slot);
 }
 
-async function createPTB(client: SuiClient, keypair: Ed25519Keypair, addresses: string[][]) {
+async function set_winner(
+	client: SuiClient,
+	keypair: Ed25519Keypair,
+	acof: Auction,
```

Comment:
The usual shortcut is cfg, not cof.

```suggestion
	acfg: Auction,
```